src/utils/PaymentsConverter.ts

```diff
@@ -65,7 +62,7 @@ export function getPaymentRequestsGetResponse(
   const datiPagamentoPA =
     iNodoVerificaRPTOutput.nodoVerificaRPTRisposta.datiPagamentoPA;
   return PaymentRequestsGetResponse.decode({
-    importoSingoloVersamento: datiPagamentoPA.importoSingoloVersamento,
+    importoSingoloVersamento: datiPagamentoPA.importoSingoloVersamento * 100,
```

Comment:
this conversion must be explicit and output a tagged integer type ie.

```ts
interface IEuroCentsTag {
  readonly kind: "IEuroCents";
}

type IntEuroCents = tag<IEuroCentsTag>()(t.Integer);

function toEuroCents(euros: number): IntEuroCents {
  return t.Integer.decode(euros * 100)
        .getOrElse(() => {  throw new Error("invalid amount"); })  as IntEuroCents;
}
```

so any function that takes this integer as input can be assured that it is an Integer
(that's because there's no integer type in javascript and you cannot enfoce it without a tagged type):

```ts
f(cents: IntEuroCents) { ... }
```

Reply:
It's validated here:
return PaymentActivationsPostResponse.decode({..})